src/tests/JIT/Methodical/doublearray/dblarray4.cs

```diff
@@ -22,6 +22,7 @@ public class DblArray4
     [Fact]
     [SkipOnCoreClr("This test is not compatible with GCStress.", RuntimeTestModes.AnyGCStress)]
     [OuterLoop]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/101284", typeof(TestLibrary.Utilities), nameof(TestLibrary.Utilities.IsNativeAot))]
```

Comment:
This is failing in some outerloop jobs with
> src/tests/JIT/Methodical/doublearray/dblarray4.cs(25,76): error CS0246: The type or namespace name 'TestLibrary' could not be found (are you missing a using directive or an assembly reference?) [/__w/1/s/src/tests/JIT/Methodical/doublearray/dblarray4_cs_d.csproj]

For example, from https://dev.azure.com/dnceng-public/public/_build/results?buildId=673058&view=logs&jobId=3725811e-4a8b-5f6a-b03b-a353ee894148&j=3725811e-4a8b-5f6a-b03b-a353ee894148&t=d59fccfb-2b13-5841-75bf-bfec705ee16a

Reply:
#102134